src/plugins/condition/utils/constants.js

```diff
@@ -2,3 +2,23 @@ export const TRIGGER = {
     ANY: 'any',
     ALL: 'all'
 };
+
+export const getStyleProp = (key, defaultValue) => {
```

Comment:
A file named `constants.js` doesn't feel like the right place for a utility function to me?

Reply:
Moved it to it's own utils file.